### DISCUSSION TITLE
How a modding team can breathe new life into Bomb Squad:

### DISCUSSION THREAD

**Discussion by HeyErfan** [efroemling/ballistica#682] [Ideas]
You can create a team on Discord for those who want to learn the BombSquad API and become modders. It would be great if you could clearly state a specific plan for becoming a modder in the team so that they know where to start (this is the most important thing and the most common question asked at the beginning) and they can check at any time to see where they are on their journey to reach their goal.

My suggestion is to start by asking each person what they want to learn, e.g. gamemod, maps, ui, etc.

Separate them based on their choice and offer them a few mods of your own suggestion and explain the initial workings to them. That's it! All that's left is for them to ask for your help while they are learning so that they can fully stand on their own two feet and carry on the work themselves. This will definitely help the development and progress of BombSquad because new modders with new creativity and new ideas are on the way. You just have to cast the hook.

**Comment by MatiasMGz** [comment]
We have the wiki to learn how to mod. But i would still like that there was a BS discord server for people who only plays it and a BS modding server, that could be better for modders in general.
